cfme/cloud/tenant.py

```diff
@@ -175,10 +176,15 @@ class TenantDetailsView(TenantView):
     sidebar = View.nested(TenantDetailsAccordion)
     entities = View.nested(TenantDetailsEntities)
 
+    view_selector = View.nested(DetailsToolBarViewSelector)
+
     @property
     def is_displayed(self):
         """Is this page currently being displayed"""
         expected_title = '{} (Summary)'.format(self.context['object'].name)
+        if self.extra.appliance.version > "5.11":
```

Comment:
I don't think is_displayed is the place to make this view selection, it should be done in `TenantDetails.step` navigation method.

`is_displayed` should not be modifying the view, just asserting against the content/state of view elements.

Reply:
thanks! I had doubts about placing this step in `is_displayed`, now it makes more sense :+1:

Reply:
fixed